src/software/simulated_tests/robot_stops_in_region.py

```diff
@@ -0,0 +1,55 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class RobotStopsInRegion(Validation):
```

Comment:
We currently arent using this is that intentional?

Reply:
this file is not used. It could be a useful feature for other verification; however, it is not tested for now.